koku/api/iam/customer_manager.py

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Management capabilities for Customer functionality."""
+
+import logging
+
+from django.core.exceptions import ValidationError
+
+from api.iam.models import Customer
+from api.iam.serializers import CustomerSerializer
+
+LOG = logging.getLogger(__name__)
+
+
+class CustomerManagerError(Exception):
+    """General Exception class for CustomerManager errors."""
+
+    def __init__(self, message):
```

Comment:
I like that this is explicit, but I think the init is unnecessary. Subclassing an Exception with a `pass` should also let you pass in your message just as you are down below.